courier/template/load_template.go

```diff
@@ -61,12 +71,39 @@ func loadBuiltInTemplate(filesytem fs.FS, name string, html bool) (Template, err
 		tpl = t
 	}
 
-	_ = cache.Add(name, tpl)
+	_ = Cache.Add(name, tpl)
 	return tpl, nil
 }
 
+func loadRemoteTemplate(ctx context.Context, d templateDependencies, url string, name string, html bool, root string) (Template, error) {
+	if t, found := Cache.Get(name); found {
+		return t.(Template), nil
+	}
+
+	f := fetcher.NewFetcher(fetcher.WithClient(d.HTTPClient(ctx)))
+
+	bb, err := f.Fetch(url)
```

Comment:
In contrast, we can definitely cache this request, so a cache with key `url` that returns `bb`

Reply:
So instead of caching and retrieving the template we cache the fetch `bytes.Buffer`?